pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -752,9 +740,28 @@ func (r *RepoSyncReconciler) mapConfigMapToRepoSyncs(obj client.Object) []reconc
 	return requests
 }
 
+func repoSyncHelmValuesFileNames(rs *v1beta1.RepoSync) []string {
```

Comment:
nit: this is only used to be wrapped with `slices.Contains`, so this function could probably be written as something like `helmValuesFileRefsContains`

Reply:
yeah, I considered that, it would have been slightly more efficient, but I thought this looked a slight bit more readable/reusable. I don't think the list will ever get big enough to actually impact performance significantly.